descwl_shear_sims/sim.py

```diff
@@ -405,6 +420,11 @@ def make_exp(
 
     exp = afw_image.ExposureF(masked_image)
 
+    # Prepare the frc
```

Comment:
What does this do?

Reply:
https://hsc.mtk.nao.ac.jp/pipedoc/pipedoc_4_e/e_tool/index.html
See the last row of  table 1 in the link above.
In HSC pipeline, (I do not know why) the photometric calibration file is called frc...

Reply:
I mean, what do these operations do?  Do they set some metadata in the exposure?  How is it retrieved etc.

Reply:
Thanks.  Do you mind putting that as a comment in the code for future reference?

Reply:
Done!!